internal/digraph/executor/docker.go

```diff
@@ -371,16 +411,36 @@ func newDocker(
 		}
 	}
 
-	pull := true
-	if p, ok := execCfg.Config["pull"]; ok {
+	pull := ""
```

Comment:
I'm not entirely sure, but could we use `bool` or `*bool` here instead?
If we go with `string`, it might break compatibility with older versions of Dagu